include/logging/log_rpc.h

```diff
@@ -90,6 +90,24 @@ void log_rpc_set_history_level(enum log_rpc_level level);
  */
 uint8_t log_rpc_get_history_usage_threshold(void);
 
+/**
+ * @brief Gets the current history usage size in bytes.
+ *
+ * This function fetches the current history usage size in bytes.
+ *
+ * @returns The current history usage size in bytes.
+ */
+size_t log_rpc_get_history_usage_size(void);
```

Comment:
How about renaming this to something like 'log_rpc_get_history_usage_current' and the other function to `log_rpc_get_history_usage_max`?